manifests/compose.pp

```diff
@@ -13,9 +13,14 @@
 #   The version of Docker Compose to install.
 #   Defaults to the value set in $docker::params::compose_version
 #
+# [*install_from*]
```

Comment:
`install_url` is probably a better name for this, as is describes that the param should be in the name, rather than relying on the description.